### ISSUE TITLE
[Question] Plan on CRAN?

### ISSUE THREAD

**Issue by dipterix** [bbuchsbaum/neuroim2#4]
Hi, is there any plan to publish your package on CRAN?

**Comment by bbuchsbaum** [comment]
I'm willing to publish on CRAN. I'll consider your question a vote in favor of developing such a plan.

**Comment by dipterix** [comment]
I'm developing 3D viewer in R to visualize brain. It's called [threeBrain](https://dipterix.github.io/threeBrain/). I'd love to use your package as reader to support AFNI/SUMA structure.

Before it's on CRAN, would you mind if I use part of your code in my package?

**Comment by bbuchsbaum** [comment]
yes, go ahead and use code with one proviso. If you enhance or add useful code, please contribute back. Also, the AFNI stuff is incomplete and probably belongs in a separate package anyway. You might create a small package to read AFNI headers that could then be used by neuroim2 down the road.
